MonoDevelop.FSharp.Tests/TestTooltipProvider.fs

```diff
@@ -84,5 +84,218 @@ type TestTooltipProvider() =
    source1  : seq<'T1>           ->
    source2  : seq<'T2>           
            -> bool"""
+        
+        signature.ToString() |> should equal expected
+        // Base Type Constraint
+
+    [<Test>]
+    member this.Formats_base_type_constraint_tooltip() =
```

Comment:
It only works on the Roslyn branch of XS. I was thinking of renaming all the tests once Roslyn is stable.